### PR TITLE
[User] Documentation of User component

### PR DESCRIPTION
**Pull request by michalmarcinkowski**
| Q             | A
| ------------- | ---
| Doc fix?      | no
| New docs?     | yes
| Fixed tickets | 

Based on https://github.com/Sylius/Sylius-Docs/pull/316

### CODE REVIEW

File: components/User/summary.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Summary
+=======
+
+Bug tracking
+------------
+
+This component uses `GitHub issues <https://github.com/Sylius/Sylius/issues>`_.
+If you have found bug, please create an issue.
```
Comment (pjedrzejewski) [inline components/User/summary.rst]: Move to using components, because it applies to all components.

**Comment by diimpp** [comment]
Would be cool to be merged asap.
Empty docs better, than nothing :)
Plus it will allow contribute.

**Comment by pjedrzejewski** [comment]
@michalmarcinkowski Yes, let's merge it even if you want to add something later. Let us know when it is ready for review.

**Comment by michalmarcinkowski** [comment]
@pjedrzejewski @diimpp ready for review. I will finish the docs as soon as I have some free time.

**Comment by diimpp** [comment]
Would be cool to have basic structure for user bundle as well.
Even fully empty one:
- Installation
- Configuration Reference

Because I think, that docs contributions are mostly done via web interface.

:+1:

**Comment by pjedrzejewski** [comment]
Thank you Michał!

**Pull request merged**